metisp/pymetis/src/pymetis/tests/test_edps.py

```diff
@@ -34,7 +34,8 @@ def inner():
 workflows = ['metis_lm_img_wkf', 'metis_lm_ifu_wkf', 'metis_pupil_imaging_wkf', 'metis_wkf']
 
 
-class DisabledTestEDPS:
+@pytest.mark.edps
+class TestEDPS:
     @pytest.mark.parametrize('workflow_name', workflows)
```

Comment:
I cannot comment on unchanged lines unfortunately, but the `workflows` should not include `metis_lm_ifu_wkf`, just `metis_ifu_wkf`

Reply:
Thanks, I spent two hours digging into this.